components/03-modules/slider/slider.js

```diff
@@ -0,0 +1,18 @@
+/* globals lory */
+
+(function() { // eslint-disable-line
```

Comment:
It might be a good idea to use named function here

Reply:
it's not required, let it for now b/c we have other function created like this i.e select, range, popup...

Reply:
You're right, following good practices is not required , but it is recommended. Using named function with IIFE is not that hard : 
```
(function alpacaSlider() { // eslint-disable-line
  'use strict';

  const sliders = [...document.querySelectorAll('.slider')];

  sliders.forEach(slider => {
    lory(slider, {
      rewind: true,
      enableMouseEvents: true,
      classNameFrame: 'slider__frame',
      classNameSlideContainer: 'slider__slides',
      classNamePrevCtrl: 'slider__prev',
      classNameNextCtrl: 'slider__next'
    });
  });
}());
```
I guess it's not a big change ? And it will make it easier for potential debugging in future.

Reply:
okay, changed